rpc/src/tests/rpc.rs

```diff
@@ -40,6 +40,7 @@ fn serve(handler: Option<MetaIoHandler<Metadata>>) -> Server<HttpServer> {
 			}
 		}),
 		1,
+        5,
```

Comment:
Incorrect indentation.

Reply:
Thanks for the review. I've corrected the indentation issue.